src/FBSettings.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+const Settings = require('react-native').NativeModules.FBSettings;
+
+module.exports = {
+  /**
+   * For iOS only, get AdvertiserTrackingEnabled status.
+   * @platform ios
+   */
+  getAdvertiserTrackingEnabled(): Promise<boolean> {
```

Comment:
Should we add a platform check here and no-op / warn on Android instead of crashing?